test/TestProjects/MgmtDiscriminator/MgmtDiscriminator.json

```
@@ -579,6 +579,7 @@
                 "cacheDuration": {
                     "description": "The duration for which the content needs to be cached. Allowed format is [d.]hh:mm:ss",
                     "type": "string",
+                    "format": "duration",
```

```suggestion
                    "format": "duration",
                    "x-ms-format": "duration-constant"
```
Based on the description, it's using the constant format not the default period format.